config/environments/development.rb

```diff
@@ -0,0 +1,66 @@
+Rails.application.configure do
```

Comment:
I missed this earlier, but this is a bit inconvenient. Development configuration should not be committed to source control, it is not the same for everyone.

We have a file called development.rb.sample where the default configuration should be put and every developer can adjust it to their local setup.

I'll make a PR to correct this.